outreachy-round-13.md

```diff
@@ -0,0 +1,106 @@
+product quality is extremely high -- you can except to learn a lot from
+disciplined code reviews by highly experienced engineers. Since Zulip is a
+group chat product, your Outreachy experience with the Zulip project will be
+highly interactive, with a real focus on teaching you the concepts and
+reasoning behind how Zulip is engineered and how to make it better.
+
+# Get to know us
+
+We have an easy-to-setup development environment, and a library of
+tasks that are great for first-time contributors
+([guide](https://github.com/zulip/zulip#contributing-to-zulip)).
+
+We'd love for you to drop by [our open livechat at
+[zulip.tabbott.net](https://zulip.tabbott.net/) or our [development
+mailing list](https://groups.google.com/forum/#!forum/zulip-devel),
+talk with us about [our
+roadmap](https://zulip.readthedocs.io/en/latest/roadmap.html) or your
+questions, and consider applying for an Outreachy internship with us!
+
+# Application tips
+
+Our Outreachy application process opens September 12 and closes
+October 17. [You'll follow the instructions
+here.](https://wiki.gnome.org/Outreachy#Application_Process)
+
+Your application should include the following:
+
+* Details on any experience you have related to the technologies that
+Zulip has, or related to our product approach.
+* Links to materials to help us evaluate your level of experience and
+  how you work, such as personal projects of yours, including any
+  existing open source or open culture contributions you've made and
+  any bug reports you've submitted to open source projects.
+* Some notes on what you are hoping to get out of your summer project.
+* A description of the project you'd like to do, and why you're excited about it.
+* Some notes on why you're excited about working on Zulip.
+* A link to the small initial contribution you did.
+
+If you're applying for a coding internship: We expect applicants to
+either have experience with the technologies relevant to their project
+or have strong programming experience.  We also expect applicants to
+be excited about learning how to do disciplined, professional software
+engineering, where they can demonstrate through reasoning and
+automated tests that their code is correct.
+
+If you're applying for a writing internship: We expect applicants to
+be skilled in writing in English for a non-academic audience, and to
+be interested in getting feedback that will help them write accurately
+and accessibly about technology.
+
+
+# Project ideas
+
+* [writing bots and integrations](https://zulip.readthedocs.io/en/latest/integration-guide.html) -- e.g., [a better IRC mirror](https://github.com/zulip/zulip/issues/249) or Jabber mirror or [Mailman integration](https://github.com/zulip/zulip/issues/959)
+* [making it easy to import data from Slack into Zulip](https://github.com/zulip/zulip/issues/908)
+* working on our [React Native mobile app](https://github.com/zulip/zulip-mobile)
```

Comment:
Add the word "iOS" for the React Native mobile app, and also add "working on our Java Android app"